### ISSUE TITLE
question about training loss and inference performance

### ISSUE THREAD

**Issue by zzw922cn** [auspicious3000/autovc#61]
Hi, thank you for your very nice work! I have rerun this project, and it has run 90K steps. the  loss_id_psnt is around 0.07. And I tried to feed into a in-domain speaker's melspec and his speaker embedding as source embedding, and another speaker's speaker embedding as target speaker embedding. Then I use GL vocoder to generate the wav, I found the voice is still of the source speaker. Is this normal? When can I perform voice conversion successfully? at what step or what's the loss_id_psnt? thank you very much!!

![image](https://user-images.githubusercontent.com/11649939/96944380-3dc4d500-150d-11eb-9c96-a0a13982a1c8.png)

**Comment by auspicious3000** [comment]
You probably need to fine-tune your bottleneck dimensions.

**Comment by zzw922cn** [comment]
Do you think I should enlarge the bottleneck dimension or decrease the bottleneck dimension?

**Comment by auspicious3000** [comment]
There's detailed information in the paper on how to tune the bottleneck.

**Comment by zzw922cn** [comment]
OK, thank you~

**Comment by ruclion** [comment]
> Do you think I should enlarge the bottleneck dimension or decrease the bottleneck dimension?

the paper said:
The first model, which we name the “too narrow” model, reduces the dimensions of C1→ and C1← from 32 to 16, and increases the downsampling factor from 32 to 128 (note that higher downsampling factor means lower temporal dimension). The second model, which we name the “too wide” model, increases the dimensions of C1→ and C1← to 256, and decreases the sampling factor to 8, and λ is set to 0

But for new dataset, how to choose the hparams? And wheather we should use DANN idea? 
Hope to communicate with you~

**Comment by innovator1311** [comment]
> Hi, thank you for your very nice work! I have rerun this project, and it has run 90K steps. the loss_id_psnt is around 0.07. And I tried to feed into a in-domain speaker's melspec and his speaker embedding as source embedding, and another speaker's speaker embedding as target speaker embedding. Then I use GL vocoder to generate the wav, I found the voice is still of the source speaker. Is this normal? When can I perform voice conversion successfully? at what step or what's the loss_id_psnt? thank you very much!!
> 
> ![image](https://user-images.githubusercontent.com/11649939/96944380-3dc4d500-150d-11eb-9c96-a0a13982a1c8.png)

@zzw922cn Call you tell me which dataset you used and the batch size of training process ? Thanks in advance !!
